java/map-loaded/README.md

```diff
@@ -4,7 +4,7 @@ This sample shows how to tell what the map's load status is. This is obtained fr
 * The map has an an initial viewpoint
 * One of the map's predefined layers has been created.
 
-A signal handler is set up on the map to handle the ```LoadStatusChangedEvent``` signal, and the status text is updated when the status changes.
+A signal handler is set up on the map to handle the `LoadStatusChangedEvent` signal, and the status text is updated when the status changes.
```

Comment:
This might have been copied verbatim from Qt - 'signals' is not correct terminology for Java/Android (the listener is set up to handle the event).